README.md

```diff
@@ -6,8 +6,8 @@
 <a href='https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA'><img src='https://img.shields.io/static/v1?label=chat&logo=slack&message=Slack&color=brightgreen' alt='Slack'  align="center"/></a>
 <a href="https://codecov.io/gh/modin-project/modin"><img src="https://codecov.io/gh/modin-project/modin/branch/master/graph/badge.svg" align="center"/></a>
 <a href="https://github.com/modin-project/modin/actions"><img src="https://github.com/modin-project/modin/workflows/master/badge.svg" align="center"></a>
-<a href="https://modin.readthedocs.io/en/0.15.1/?badge=0.15.1"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=0.15.1" align="center"></a>
-<a href="https://pypi.org/project/modin/0.15.1/"><img src="https://img.shields.io/badge/pypi-0.15.1-blue.svg" alt="PyPI version" align="center"></a>
+<a href="https://modin.readthedocs.io/en/0.15.3/?badge=0.15.3"><img alt="" src="https://readthedocs.org/projects/modin/badge/?version=0.15.3" align="center"></a>
+<a href="https://pypi.org/project/modin/0.15.3/"><img src="https://img.shields.io/badge/pypi-0.15.3-blue.svg" alt="PyPI version" align="center"></a>
```

Comment:
does this mean this step was missed in 0.15.2?

Reply:
Yes, we missed that part last time.